oemoflex/postprocessing.py

```diff
@@ -8,11 +9,53 @@
 from oemof.solph import EnergySystem, Bus, Sink
 from oemof.tabular import facades
 import oemof.tabular.tools.postprocessing as pp
+from oemof.tools.economics import annuity
 from oemoflex.helpers import delete_empty_subdirs, load_elements
 from oemoflex.preprocessing import get_parameter_values
 
+from oemoflex.facades import TYPEMAP
+
 basic_columns = ['region', 'name', 'type', 'carrier', 'tech']
 
+FlexMex_Parameter_Map = {
```

Comment:
As a function? Add 'input' in the name.

Reply:
This could be used in preprocessing as well, no?